src/Form/Input.js

```diff
@@ -221,6 +278,11 @@ class Input extends Component {
     this.ref.current.blur();
   }
 
+  handleClear = () => {
+    this.setState({ value: '' });
+    this.props.onChange(this.inputRef.current.name, '');
```

Comment:
I actually just removed that `handleClear` method. It's not needed in the Input since that's specific to another component. I believe this duplication is taken care of as well with this removal.